bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/model/VerisureAlarms.java

```diff
@@ -0,0 +1,273 @@
+            this.statusType = statusType;
+        }
+
+        public @Nullable String getDate() {
+            return date;
+        }
+
+        public void setDate(@Nullable String date) {
+            this.date = date;
+        }
+
+        public @Nullable String getName() {
+            return name;
+        }
+
+        public void setName(@Nullable String name) {
+            this.name = name;
+        }
+
+        public @Nullable String getChangedVia() {
+            return changedVia;
+        }
+
+        public void setChangedVia(@Nullable String changedVia) {
+            this.changedVia = changedVia;
+        }
+
+        public boolean getAllowedForFirstLine() {
+            return allowedForFirstLine;
+        }
+
+        public void setAllowedForFirstLine(boolean allowedForFirstLine) {
+            this.allowedForFirstLine = allowedForFirstLine;
+        }
+
+        public boolean getAllowed() {
+            return allowed;
+        }
+
+        public void setAllowed(boolean allowed) {
+            this.allowed = allowed;
+        }
+
+        public List<Object> getErrorCodes() {
+            return errorCodes;
+        }
+
+        public void setErrorCodes(List<Object> errorCodes) {
+            this.errorCodes = errorCodes;
+        }
+
+        public @Nullable String getTypename() {
+            return typename;
+        }
+
+        public void setTypename(@Nullable String typename) {
+            this.typename = typename;
+        }
+
+        @Override
+        public String toString() {
+            return new ToStringBuilder(this).append("type", type).append("statusType", statusType).append("date", date)
+                    .append("name", name).append("changedVia", changedVia)
+                    .append("allowedForFirstLine", allowedForFirstLine).append("allowed", allowed)
+                    .append("errorCodes", errorCodes).append("typename", typename).toString();
+        }
+
+        @Override
+        public int hashCode() {
+            return new HashCodeBuilder().append(allowed).append(statusType).append(typename).append(name)
+                    .append(allowedForFirstLine).append(errorCodes).append(date).append(type).append(changedVia)
+                    .toHashCode();
+        }
+
+        @Override
+        public boolean equals(@Nullable Object other) {
+            if (other == this) {
+                return true;
+            }
+            if (!(other instanceof ArmState)) {
+                return false;
+            }
+            ArmState rhs = ((ArmState) other);
+            return new EqualsBuilder().append(allowed, rhs.allowed).append(statusType, rhs.statusType)
+                    .append(typename, rhs.typename).append(name, rhs.name)
+                    .append(allowedForFirstLine, rhs.allowedForFirstLine).append(errorCodes, rhs.errorCodes)
+                    .append(date, rhs.date).append(type, rhs.type).append(changedVia, rhs.changedVia).isEquals();
+        }
```

Comment:
Are equals,hashcode, and toString even used for your DTO classes? If not I suggest removing them.

Reply:
They are used for detecting changes to control of the Session HashMap with these things should be updated. toString is used for debugging purposes.